test/extended/clusterup.sh

```diff
@@ -0,0 +1,245 @@
+    export HOME="${test_home}"
+    pushd "${HOME}" &> /dev/null
+    os::log::info "Using ${HOME} as home directory"
+    ${funcname}
+    popd &> /dev/null
+    export HOME="${global_home}"
+
+    os::log::info "cluster up -- ${test}: ok"
+    os::test::junit::declare_suite_end
+}
+
+function os::test::extended::clusterup::verify_router_and_registry () {
+    os::cmd::expect_success "oc login -u system:admin"
+    os::cmd::expect_success_and_text "oc get svc -n default" "docker-registry"
+    os::cmd::expect_success_and_text "oc get svc -n default" "router"
+	os::cmd::try_until_text "oc get endpoints docker-registry -o jsonpath='{ .subsets[*].ports[?(@.name==\"5000-tcp\")].port }' -n default" "5000" $(( 10*minute )) 1
+	os::cmd::try_until_text "oc get endpoints router -o jsonpath='{ .subsets[*].ports[?(@.name==\"80-tcp\")].port }' -n default" "80" $(( 10*minute )) 1
+    os::cmd::expect_success "oc login -u developer"
+}
+
+function os::test::extended::clusterup::verify_image_streams () {
+    os::cmd::try_until_text "oc get is -n openshift ruby -o jsonpath='{ .status.tags[*].tag }'" "latest" $(( 10*minute )) 1
+}
+
+function os::test::extended::clusterup::verify_ruby_build () {
+    os::cmd::expect_success "oc new-app https://github.com/openshift/ruby-hello-world.git"
+    os::cmd::try_until_text "oc get builds -l app=ruby-hello-world -o jsonpath='{ .items[*].status.phase }'" "Complete" $(( 10*minute )) 1
+}
+
+function os::test::extended::clusterup::verify_persistent_volumes () {
+    os::cmd::expect_success "oc login -u system:admin"
+    os::cmd::expect_success_and_text "oc get jobs -n default" "persistent-volume-setup"
+    os::cmd::try_until_text "oc get job persistent-volume-setup -n default -o jsonpath='{ .status.succeeded }'" "1" $(( 10*minute )) 1
+    os::cmd::expect_success "oc get pv/pv0100"
+    os::cmd::expect_success "oc login -u developer"
+}
+
+function os::test::extended::clusterup::verify_metrics () {
+    os::cmd::expect_success "oc login -u system:admin"
+    os::cmd::expect_success_and_text "oc get pods -n openshift-infra" "metrics-deployer"
+    os::cmd::try_until_text "oc get pods -n openshift-infra -l job-name=metrics-deployer-pod" "Completed" $(( 20*minute )) 2
+	os::cmd::try_until_text "oc get endpoints hawkular-metrics -o jsonpath='{ .subsets[*].ports[?(@.name==\"443-tcp\")].port }' -n openshift-infra" "443" $(( 10*minute )) 1
+    os::cmd::expect_success "oc login -u developer"
+}
+
+function os::test::extended::clusterup::verify_logging () {
+    os::cmd::expect_success "oc login -u system:admin"
+    os::cmd::expect_success_and_text "oc get pods -n logging" "logging-deployer"
+    os::cmd::try_until_text "oc get pods -n logging -l logging-infra=deployer" "Completed" $(( 20*minute )) 2
+	os::cmd::try_until_text "oc get endpoints logging-kibana -o jsonpath='{ .subsets[*].ports[?(@.name==\"3000-tcp\")].port }' -n logging" "3000" $(( 10*minute )) 1
+    os::cmd::expect_success "oc login -u developer"
+}
```

Comment:
it needs to use a new origin image when running.  the requirement to have a policybinding before creating a binding was recently lifted.

I would have expected this test to run using the local origin image anyway, though (--version=latest)